client/src/components/QuestionsAndAnswers/index.jsx

```diff
@@ -50,7 +51,7 @@ const QuestionsAndAnswers = () => (
     </div>
     <div className={styles.buttonContainer}>
       <div className={styles.moreAnsweredQuestionsButton}>More Answered Questions</div>
-      <div className={styles.addAQuestionButton}>Add A Question</div>
+      <div className={styles.addAQuestionButton}><AddAQuestion /></div>
```

Comment:
For the future, if you were to replace the surrounding div with the component, you could cut down on the number of places you need to style and have better separations of concerns

```suggestion
      <AddAQuestion />
```